SERVER.md

```diff
@@ -0,0 +1,24 @@
+# VotingWorks BMD Server
+
+When running on an actual voting machine in localhost mode, you should use this
+simple server.
+
+The instructions here are for Ubuntu Linux 18.04
+
+## Install Requisite Software
+
+```
+sudo apt install python3
```

Comment:
nit: When sudo is used, I've been conditioned to always look for a less forceful path.

Reply:
it's a little tricky to not use `sudo` in this particular case. I'll keep an eye out for how to find an alternative to this in the future.